plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/messages.properties

```diff
@@ -1,12 +1,12 @@
+BUTTON_ACCOUNTS_PANEL_ADD_ACCOUNT=Add Account
+BUTTON_ACCOUNTS_PANEL_LOGOUT=Sign Out
 LOGIN_ERROR_CANNOT_OPEN_BROWSER=Error opening a system browser. Signing in requires an open browser.
 LOGIN_ERROR_DIALOG_MESSAGE=An error occurred while trying to sign in.
 LOGIN_ERROR_DIALOG_TITLE=Sign in Failed
 LOGIN_ERROR_LOCAL_SERVER_RUN=Error listening for authorization code:\n\n
-LOGIN_MENU_LOGGED_IN=Sign out from Google
+LOGIN_MENU_LOGGED_IN=Manage Google Accounts...
 LOGIN_MENU_LOGGED_OUT=Sign in to Google...
 LOGIN_PROGRESS_DIALOG_MESSAGE=Please complete signing in using the external browser.
```

Comment:
delete "Please" http://ux.stackexchange.com/questions/25500/why-avoid-the-word-please-in-error-messages